metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -391,12 +399,18 @@ def make_flow(
             "Argo Workflows requires --datastore=s3 or --datastore=azure or --datastore=gs"
         )
 
-    if (notify_on_error or notify_on_success) and not notify_slack_webhook_url:
+    if (notify_on_error or notify_on_success) and not (
+        notify_slack_webhook_url or notify_pagerduty_integration_key
+    ):
         raise MetaflowException(
-            "Slack notifications require specifying an incoming Slack "
+            "Workflow notifications require specifying a notification provider.\n\n"
```

Comment:
`Notifications require specifying an incoming Slack webhook url via --notify-slack-webhook-url or Pager Duty v3 webhook url via --notify-pager-duty-webhook-url.\n
If you would like to set up notifications for your Slack workspace, follow the instructions at https://api.slack.com/messaging/webhooks to generate a webhook url.\n For notifications through Pager Duty, generate an integration key by following the instructions at https://support.pagerduty.com/docs/services-and-integrations#create-a-generic-events-api-integration"`

Reply:
Second line should probably instead be
`Pager Duty Events API V2 integration key via --notify-pager-duty-integration-key.\n If you would like to set up `

From what I glanced, the PagerDuty webhook support is for providing a webhook subscription (own endpoint for Pagerduty to send data to), which is why it is not used here. 

`--notify-pager-duty-integration-key` is the only required value, as the Events API V2 is a static endpoint.